app/views/giving/transactions/new.html.haml

```diff
@@ -0,0 +1,67 @@
+- @title = t('giving.heading')
+
+- content_for :js do
```

Comment:
this script will need a new home. 
It's always a good idea to split javascript from html.